app/lib/features/tasks/pages/task_list_details_page.dart

```diff
@@ -88,9 +113,19 @@ class _TaskListPageState extends ConsumerState<TaskListDetailPage> {
     return Column(
       crossAxisAlignment: CrossAxisAlignment.start,
       children: [
-        Text(
-          taskListData.description()!.body(),
-          style: Theme.of(context).textTheme.labelMedium,
+        GestureDetector(
+          onTap: () {
+            showEditDescriptionSheet(taskListData);
+          },
+          child: taskListData.description()!.formattedBody() != null
+              ? RenderHtml(
+                  text: taskListData.description()!.formattedBody()!,
```

Comment:
see above. also the idea to just make that a Widget itself given a `MsgContent` as input...?